src/components/organisms/PageHeader/ProjectSelection.tsx

```diff
@@ -0,0 +1,229 @@
+import {setCreateProject, setDeleteProject, setOpenProject} from '@redux/reducers/appConfig';
+import {openCreateProjectModal} from '@redux/reducers/ui';
+import {activeProjectSelector, settingsSelector} from '@redux/selectors';
+
+import FileExplorer from '@components/atoms/FileExplorer';
+
+import {useFileExplorer} from '@hooks/useFileExplorer';
+
+import {
+  StyledDeleteOutlined,
+  StyledFolderOpenOutlined,
+  StyledProjectButton,
+  StyledProjectFolderAddOutlined,
+  StyledProjectFolderOpenOutlined,
+  StyledProjectFormatPainterOutlined,
+  StyledProjectMenu,
+  StyledProjectTableActions,
+  StyledProjectsDropdown,
+  StyledProjectsMenuActionsContainer,
+  StyledProjectsMenuContainer,
+  StyledSearch,
+} from './Styled';
+
+const ProjectSelection = () => {
+  const dispatch = useAppDispatch();
+
+  const projects: Project[] = useAppSelector(state => state.config.projects);
+  const activeProject = useSelector(activeProjectSelector);
+  const {isClusterSelectorVisible} = useAppSelector(settingsSelector);
+  const [isDropdownMenuVisible, setIsDropdownMenuVisible] = useState(false);
+  const [searchText, setSearchText] = useState('');
+  const [filteredProjects, setFilteredProjects] = useState<Project[]>([]);
+
+  const {openFileExplorer, fileExplorerProps} = useFileExplorer(
+    ({folderPath}) => {
+      if (folderPath) {
+        dispatch(setCreateProject({rootFolder: folderPath}));
+      }
+    },
+    {isDirectoryExplorer: true}
+  );
+
+  useEffect(() => {
+    if (searchText) {
+      setFilteredProjects(
+        _.filter(
+          projects,
+          (p: Project) =>
+            _.includes(p.name?.toLowerCase(), searchText.toLowerCase()) ||
+            _.includes(p.rootFolder.toLowerCase(), searchText.toLowerCase())
+        )
+      );
+    }
+  }, [searchText, projects]);
+
+  const handleProjectSearch = (e: any) => {
+    setSearchText(e.target.value);
+  };
+
+  const handleProjectChange = (project: Project) => {
+    setIsDropdownMenuVisible(false);
+    setTimeout(() => dispatch(setOpenProject(project.rootFolder)), 400);
+  };
+
+  const handleCreateProject = (fromTemplate: boolean) => {
+    setIsDropdownMenuVisible(false);
+    dispatch(openCreateProjectModal({fromTemplate}));
+  };
+
+  const handleDeleteProject = (project: Project) => {
+    dispatch(setDeleteProject(project));
+  };
+
+  // const handleCopyProject = (project: Project) => {
+  //   setIsDropdownMenuVisible(false);
+  // };
+
+  // const handleEditProject = (project: Project) => {
+  //   setIsDropdownMenuVisible(false);
+  // };
+
+  const getRelativeDate = (isoDate: string | undefined) => {
+    if (isoDate) {
+      return DateTime.fromISO(isoDate).toRelative();
+    }
+    return '';
+  };
+
+  const projectMenu = () => {
+    return (
+      <StyledProjectMenu>
+        <StyledProjectsMenuContainer>
+          <StyledSearch placeholder="Search" value={searchText} onChange={handleProjectSearch} />
```

Comment:
We should add tooltips to these 4:
- Search: "Search for project by name"
- Browse: "New project from existing folder"
- Empty project: "New Empty Project"
- Project from template: "New Project from Template"